care/facility/migrations/0315_auto_20220905_1008.py

```diff
@@ -0,0 +1,57 @@
+# Generated by Django 2.2.11 on 2022-09-05 04:38
+
+import enum
+from django.db import migrations, models
+
+
+class OccupationEnum(enum.Enum):
+    STUDENT = 1
+    MEDICAL_WORKER = 2
+    GOVT_EMPLOYEE = 3
+    PRIVATE_EMPLOYEE = 4
+    HOME_MAKER = 5
+    WORKING_ABROAD = 6
+    OTHERS = 7
+
+
+OccupationChoices = [(item.value, item.name) for item in OccupationEnum]
+occupation_list = []
+
+
+def get_occupation(apps, schema_editor):
+    PatientRegistration = apps.get_model("facility", "PatientRegistration")
+    patients = PatientRegistration.objects.all()
+
+    for patient in patients:
+        occupation_list.append(
+            {
+                "patient": patient,
+                "occupation": OccupationChoices[patient.meta_info.occupation],
```

Comment:
Need to handle cases where meta info is not present, since its a nullable field.